packages/google_maps_flutter/ios/Classes/JsonConversions.m

```diff
@@ -37,11 +37,11 @@ + (NSArray*)positionToJson:(CLLocationCoordinate2D)position {
 }
 
 + (UIColor*)toColor:(NSNumber*)numberColor {
-  long value = [numberColor longValue];
+  unsigned long value = [numberColor unsignedLongValue];
   return [UIColor colorWithRed:((float)((value & 0xFF0000) >> 16)) / 255.0
                          green:((float)((value & 0xFF00) >> 8)) / 255.0
                           blue:((float)(value & 0xFF)) / 255.0
-                         alpha:1.0];
+                         alpha:((float)((value & 0xFF000000) >> 24)) / 255.0];
```

Comment:
How does this change affect people who are already using some color. Since it's signed i'm guessing `value & 0xFF000000` will default to `FF` which should still retain the existing semantics of alpha being `1.0`. Could you confirm that behavior for me?

Reply:
Yes, it will retain the existing semantics. The alpha channel already works on Android, so users are presumably already using Colors with the alpha channel set to `FF`, and this will just evaluate to `alpha: 1.0`.